pkg/plugins/golang/v3/api.go

```diff
@@ -182,6 +186,16 @@ func (p *createAPISubcommand) Scaffold(fs machinery.Filesystem) error {
 }
 
 func (p *createAPISubcommand) PostScaffold() error {
+
+	// Update the makefile to allow generate Webhooks to ensure backwards compatibility
+	// todo: it should be removed for go/v4
+	// nolint:lll,gosec
+	if p.resource.API.CRDVersion == "v1beta1" {
+		if err := applyScaffoldCustomizationsForVbeta1(); err != nil {
```

Comment:
To ensure backwards compatibility:

- If the flags crd-version or webhook-version be informed with v1beta1 the makefile and go.mod are updated, and the user informed
- Marke as deprecates the flags crd-version or webhook-version since we probably will remove it in future versions.

We cannot remove the option for go/v3. That would be a breaking change and we will allow to do that only when we create go/v4.